system-addon/content-src/lib/link-menu-options.js

```diff
@@ -71,15 +71,15 @@ export const LinkMenuOptions = {
       action_position: index
     })
   }),
-  DeleteUrl: (site, index, eventSource) => ({
+  DeleteUrl: (site, index, eventSource, value) => ({
```

Comment:
How about other menu options here? Such as `BLOCK`. Although I am not sure if we wanted to do this for all of them?

Reply:
BLOCK defines a user event here
https://github.com/mozilla/activity-stream/blob/f7bbd56c77ac849b5c6f5c702d63753c1b5e96bf/system-addon/content-src/lib/link-menu-options.js#L59
Which is handled here
https://github.com/mozilla/activity-stream/blob/f7bbd56c77ac849b5c6f5c702d63753c1b5e96bf/system-addon/content-src/components/LinkMenu/LinkMenu.jsx#L23-L28
So you should see the same topsite information as for the others. Delete was "special" and needed a different change.

Reply:
Oh, "DELETE" is a special case. So siteInfo will be attached to all the events then, correct?

Reply:
Yes